host/class/uac/usb_host_uac/uac_host.c

```diff
@@ -803,6 +809,21 @@ static esp_err_t uac_host_interface_add(uac_device_t *uac_device, uint8_t iface_
                 iface_alt->ep_addr = ep_desc->bEndpointAddress;
                 iface_alt->ep_mps = ep_desc->wMaxPacketSize;
                 iface_alt->ep_attr = ep_desc->bmAttributes;
+                if (is_full_speed && ep_desc->bInterval != 1) {
+                    // Full-Speed isochronous endpoints must have bInterval = 1
+                    uint8_t *_bInterval = (uint8_t *) & (ep_desc->bInterval);
+                    // Check if we can modify the bInterval value
+                    if (esp_ptr_in_dram(_bInterval) || esp_ptr_in_diram_dram(_bInterval)
+#if CONFIG_SPIRAM
+                            || esp_ptr_in_psram(_bInterval)
+#endif
+                       ) {
+                        ESP_LOGW(TAG, "UAC Full-Speed device, Endpoint %d, bInterval %d, set to 1", USB_EP_DESC_GET_EP_NUM(ep_desc), ep_desc->bInterval);
+                        *_bInterval = 1;
+                    } else {
+                        ESP_LOGW(TAG, "UAC Full-Speed device, Endpoint %d, bInterval %d, can't set to 1", USB_EP_DESC_GET_EP_NUM(ep_desc), ep_desc->bInterval);
+                    }
+                }
```

Comment:
This is a very risky, non-standard way that deserves explanation comment.

BTW, have you found the underlaying problem? If the device needs bInterval=2 why doesn't it work correctly with 2ms polling?